app/src/main/kotlin/at/bitfire/davdroid/servicedetection/CollectionListRefresher.kt

```diff
@@ -418,4 +421,11 @@ class CollectionListRefresher @AssistedInject constructor(
                 false
         }
     }
+
+    private fun isHomeSetPersonal(davResponse: Response): Boolean {
```

Comment:
The method checks whether a response is personal by evaluating Owner.

Section 5.1 of RFC 3744 says that `DAV:owner` can be defined on any resource. So I'd name it `isPersonal` because it's not related to homesets.

Also maybe we should return

- `true` if the owner matches a principal,
- `false` if the owner doesn't match a principal,
- `null` if owner and/or principal is not set / unknown.

Then we can leave the decision how we treat the unknown state to the caller.

(+ KDoc)